lab_3_ann_retriever/main.py

```diff
@@ -104,6 +105,17 @@ def tokenize(self, text: str) -> list[str] | None:
 
         In case of corrupt input arguments, None is returned.
         """
+        if not isinstance(text, str):
+            return None
+        punctuation = ("!'\"#$%&()*+,-./:;<=>?@[]^_`{|}~1234567890"
```

Comment:
is numbers a pucntuation too?

Reply:
no, but the task asks us to remove digits too